README.md

```diff
@@ -71,15 +71,15 @@ Like `npm start`, it runs `bower install` to install and/or update the framework
 To make the production code available to the Kubernetes api server, run this command from the top level directory:
 
 ```
-hack/build-ui.sh
+hack/build-ui.sh dashboard
```

Comment:
I don't know what the vision for this repo is, but a lot of this readme and file comments don't really make sense outside of the kubernetes repo. E.g. here: there is no hack/build-ui.sh